lib/node_modules/@stdlib/math/base/special/minmax/README.md

```diff
@@ -2,7 +2,7 @@
 
 @license Apache-2.0
 
-Copyright (c) 2018 The Stdlib Authors.
+Copyright (c) 2024 The Stdlib Authors.
```

Comment:
Use the current year as copyright year only when creating new packages. If you are modifying functionality leave the copyright year as is.
```suggestion
Copyright (c) 2018 The Stdlib Authors.
```